src/web/metrics.rs

```diff
@@ -92,6 +92,12 @@ impl<'a> RenderingTimesRecorder<'a> {
 
     fn record_current(&mut self) {
         if let Some(current) = self.current.take() {
+            #[cfg(test)]
+            log::debug!(
+                "rendering time - {}: {:?}",
```

Comment:
I added this while trying to figure out why I was getting timeouts from the two reqwest clients deadlocking, but I thought it could be useful to leave in.